modules/ROOT/pages/anypoint-mq/anypoint-mq-connector-reference.adoc

```diff
@@ -17,19 +23,19 @@ Anypoint MQ is a multitenant, cloud messaging service that enables customers to
+| Polling Time a| Number |  When prefetch is disabled, the polling time defines how often the subscriber should check for new messages. By default, this checks for messages every one second (only if the prefetch is disabled). The value of this parameter is defined in milliseconds. |  1000 |
+| Max Redelivery a| Number |  Number of redeliveries to accept for any given message. When the message redelivery count is higher than this threshold, then the message is ignored by the subscriber and not dispatched to the flow. For the message to be removed from the queue, the queue should be configured with a dead letter queue from the admin console. To avoid a poison message, the default value is set to -1 (infinite). |  -1 |
+| Fetch Size a| Number |  Max number of messages to receive on each request. The response can contain fewer messages than this number depending on the load of the target queue. This number also affects the local buffer of the prefetched messages. Setting this fetchSize to zero  disables message prefetch. The prefetch buffer has a target size of three times the fetchSize, meaning that requests to retrieve messages continue as long as the local buffer is not full. Only in the case of a slow consumer, where the Mule Runtime has no more threads available to process the request, is the buffer limit going to be reached. Enable prefetch for maximum throughput, in cases where high performance on message delivery is required. In cases where controlled message retrieval is required, you can disable prefetch and use the fixed polling configuration. By default this tries to fetch the max number of 10 messages per request. |  10 |
 | Fetch Timeout a| Number |  Maximum duration in milliseconds to wait for the required amount of messages. If this duration elapses, the response is sent with as many messages as received during the period. |  1000 |
-| Frequency a| Number |  Duration in milliseconds to execute the retrieve operation when the prefetch queue is not empty. | 5000 |
+| Frequency a| Number |  Fixed frequency in milliseconds at which the retrieval of messages os attempted when there are no more messages to process locally. In the case were a queue is empty and all the messages retrieved were already processes, this frequency defines how long to wait before attempting to retrieve more messages. By default, this frequency is 5 seconds (5000 milliseconds). |  5000 |
```

Comment:
Fixed frequency in milliseconds at which the retrieval of messages **os** attempted when there are no more messages to process locally. In the case were a queue is empty and all the messages retrieved were already **processes**

is
processed (?)